charts/app-config/values-integration.yaml

```diff
@@ -1260,6 +1260,16 @@ govukApplications:
             secretKeyRef:
               name: govuk-chat-rabbitmq
               key: RABBITMQ_URL
+        - name: GDS_SSO_OAUTH_ID
+          valueFrom:
+            secretKeyRef:
+              name: govuk-chat-oauth
+              key: GDS_SSO_OAUTH_ID
```

Comment:
```suggestion
              name: signon-app-govuk-chat
              key: oauth_id
```

Reply:
The name shouldn't change as i think it's a convention to use `app-name + secret` https://docs.publishing.service.gov.uk/kubernetes/manage-app/manage-secrets/#defining-an-externalsecret-in-kubernetes 

but I can change the key name as you suggested 👍

Reply:
It's just that all the other apps have ones prefixed with signon-app as per: https://github.com/alphagov/govuk-helm-charts/blob/2faad1e20782fa4a567a395dbd3e0ad898b85375/charts/app-config/values-integration.yaml#L122-L131

Reply:
hmm good point, they are getting them from somewhere else